packages/py-calc/0.1.0/README.md

```diff
@@ -0,0 +1,9 @@
+This is a Calculator Implementation with regex matches and Python print function. 
+
+Usage: (1+2+3.4/5)= 
+Where: 
+- The parenthesis "(", ")" and the equal symbol "=" are the matches required
+- The formula between the parenthesis is the calculation which will be performed. You can do arithmetic calculation which includes multiplication, division, addition and subtraction.
```

Comment:
and more (equality `==`, modulo `%`, exponentiation `**`, floor division `//` etc.).